refarch-integrations/refarch-s3-integration/pom.xml

```diff
@@ -19,8 +19,12 @@
     <modules>
         <module>refarch-s3-integration-core</module>
         <module>refarch-s3-integration-starter</module>
-        <module>refarch-s3-integration-service</module>
-        <module>refarch-s3-integration-client</module>
-        <module>refarch-s3-integration-client-starter</module>
+        <module>refarch-s3-integration-client-core</module>
```

Comment:
Maybe `s3-integration-core-client` would be a better name?

Reply:
I still like the current one a bit more with the default naming which indicates that the prefix `core` needs to be configured or extend.

Reply:
Ok fine. :)